.github/workflows/deploy-gh-pages.yml

```diff
@@ -1,9 +1,10 @@
-name: Deploy to GitHub Pages
+name: Sphinx Build
```

Comment:
Again, I don't think we should let the perfect be the enemy of the good and actively reject a change that clarifies something causing active confusion _right now_ (potentially not only for PEP editors, but authors as well), just because we _might_ change it at some point in the future. At least in the context of running on each PR, which is likely how 95% of people are going to actually see it (especially those besides you), calling the whole workflow "Deploy to GitHub pages" when that is actively confusing as that is the one thing it _doesn't_ do (PEP authors might assume, for instance, that it actually deploys their PEPs live somewhere), and also not being clear what build system is involved (one might assume the same one in the `Build` job because of the very confusing names).

Again, I find it difficult to argue that clarifying this, even imperfectly, is not an improvement over the status quo, and if you feel its not worth spending more effort on, the least-effort course of action for all involved is going with the above for now.